Engine/Game/Assets/Scripts/GameManager.h

```diff
@@ -20,6 +21,7 @@ class GameManager : public Script
     void Update();
 
     GameObject* GetPlayer() { return mPlayer; };
+    AudioManager* GetAudio() {return mAudioManager;};
     HudController* GetHud() { return mHudController; };
     bool UsingController() { return mController; }
```

Comment:
Second const is missing. like "GameObject* GetPlayer() **const** { return mPlayer; };"